### PR TITLE
make the example running well on AWS

### PR DESCRIPTION
**Pull request by Yancey0623**

### CODE REVIEW

File: doc/install.md
Comment (typhoonzero) [inline doc/install.md:20]: How about telling readers that if they are using windows, use a docker environment to achieve this?

File: doc/install.md
Comment (typhoonzero) [inline doc/install.md]: launching

File: doc/install.md
Comment (Yancey0623) [reply doc/install.md:20]: The user can achieve a binary file adapted windows, will add the section.

File: doc/install.md
Comment (Yancey0623) [reply doc/install.md]: Done.

File: doc/install.md
Comment (typhoonzero) [inline doc/install.md:61]: Is this nessesary or optional?

File: example/examplejob.yaml
Comment (typhoonzero) [inline example/examplejob.yaml]: image name?

File: example/train_ft.py
Comment (typhoonzero) [inline example/train_ft.py]: this use only one core.

**Review by typhoonzero — COMMENTED** [review]
LGTM, I can help test this out after it's merged.

File: doc/install.md
Comment (Yancey0623) [reply doc/install.md:61]: It's a necessary and important one.

File: example/examplejob.yaml
Comment (Yancey0623) [reply example/examplejob.yaml]: Done, changed to the official image name.

File: example/train_ft.py
Comment (Yancey0623) [reply example/train_ft.py]: Done.

**Review by typhoonzero — APPROVED** [review]

**Pull request merged**
